WalletWasabi.Gui/CommandLine/MixerCommand.cs

```diff
@@ -49,6 +49,11 @@ public override async Task<int> InvokeAsync(IEnumerable<string> args)
 			}
 			catch (Exception ex)
 			{
+				if (!(ex is OperationCanceledException))
```

Comment:
@lontivero I removed the Environment.Exit command from the Global.Initnowallet. In case of exception the ex bubbling up until the top. Please take a look.